connman/plugins/sailfish_wifi.c

```diff
@@ -42,6 +42,7 @@
 #include <errno.h>
 #include <sys/socket.h>
 #include <linux/if.h>
+#include <math.h>
```

Comment:
Drop this too now that floor() is no longer used.

Reply:
True. Fixed with 15bb6a3b2a12ffee61392387acb586a97a807f98